modules/qna/src/views/full/Components/QnA.tsx

```diff
@@ -48,17 +48,17 @@ const QnA: FC<Props> = props => {
     bp
   } = props
   const [showRedirectToFlow, setShowRedirectToFlow] = useState(!!(data.redirectFlow || data.redirectNode))
-  let questions = data.questions[contentLang] || ['']
-  let answers = data.answers[contentLang] || ['']
-  const refQuestions = (contentLang !== defaultLanguage && data.questions[defaultLanguage]) || ['']
-  const refAnswers = (contentLang !== defaultLanguage && data.answers[defaultLanguage]) || ['']
+  let questions = data.questions[contentLang]
+  let answers = data.answers[contentLang]
+  const refQuestions = contentLang !== defaultLanguage && data.questions[defaultLanguage]
+  const refAnswers = contentLang !== defaultLanguage && data.answers[defaultLanguage]
 
-  if (refQuestions && refQuestions.length > questions.length) {
-    questions = [...questions, ...Array(refQuestions.length - questions.length).fill('')]
+  if (refQuestions?.length > questions?.length || (!questions?.length && refQuestions?.length)) {
```

Comment:
technically questions will always ne defined because you set it to [''] while passing as props. But here is no such thing as too much validation right